### ISSUE TITLE
Expedition Level Permissions

### ISSUE THREAD

**Issue by ewingrj** [biocodellc/geome-ui#225]
This has been requested a few times...

Expeditions can have members, w/ read & write permissions. Similar to project members.

Would resolve following feedback about photos:

> In order to upload pictures I also had to be on Chris's account because he is the owner of the project, so it would be useful to be granted access to uploading files.

**Comment by jdeck88** [comment]
Yes, this came up again today with Chris Meyer as well as last week with Niamh and Dan....

**Comment by jdeck88** [comment]
... and just came up again in another context just now... turning out to be a pretty big issue.

**Comment by ewingrj** [comment]
a quick workaround has been implemented where a project can disable expedition access checks, allowing any project member to upload to any expedition.

it would still be good to implement finer grained expedition control at some point
